lms/templates/discussion/_filter_dropdown.html

```diff
@@ -34,19 +35,19 @@
     </li>
 </%def>
 
-<div class="forum-nav-browse-menu-wrapper" style="display: none">
+<div class="forum-nav-browse-menu-wrapper" style="display: none" role="combobox" aria-label="${_("Discussion thread list")}" aria-expanded="true" aria-owns="discussion_topics_listbox" aria-haspopup="listbox">
```

Comment:
I'm never sure about the terminology, but I don't think this is a thread list. Isn't it a topic list, since we say "filter topics"? Sometimes it is a post list too. @catong what would you advise the aria-label should be.